bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/config/project/CodeCheckerProject.java

```diff
@@ -147,6 +166,92 @@ private void updateProjectRelated() {
         }
     }
 
+    /**
+     * Merges a build log to an other.
+     * 
+     * @param newLogFile
+     *            this file will be merged into master_compilation_commands.json.
+     */
+    public void mergeCompilationLog() {
+        if (!Files.exists(masterCompCmd, LinkOption.NOFOLLOW_LINKS))
+            try {
+                Files.copy(tempCompCmd, masterCompCmd, StandardCopyOption.REPLACE_EXISTING);
+                return;
+            } catch (IOException e1) {
+                Logger.log(IStatus.ERROR, "Couldn't copy master log file.");
+            }
+
+        Set<ComilationCommand> ccmds = null;
+        Set<ComilationCommand> master_ccmds = null;
+
+        Gson gson = new GsonBuilder().disableHtmlEscaping().create();
+        try (JsonReader reader = new JsonReader(Files.newBufferedReader(tempCompCmd))) {
+            ccmds = gson.fromJson(reader, new TypeToken<Set<ComilationCommand>>() {
+            }.getType());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        try (JsonReader reader = new JsonReader(Files.newBufferedReader(masterCompCmd))) {
+            master_ccmds = gson.fromJson(reader, new TypeToken<Set<ComilationCommand>>() {
+            }.getType());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        if (ccmds != null)
+            master_ccmds.addAll(ccmds);
+
+        try (JsonWriter writer = new JsonWriter(
+                Files.newBufferedWriter(masterCompCmd, StandardOpenOption.TRUNCATE_EXISTING))) {
+            writer.setIndent("\t");
+            gson.toJson(master_ccmds, new TypeToken<Set<ComilationCommand>>() {
+            }.getType(), writer);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        deleteTemporaryLogFile();
+    }
+
+    /**
+     * Delete the specified logfile after the analysis.
+     */
+    public void deleteTemporaryLogFile() {
+        try {
+            Files.delete(tempCompCmd);
+        } catch (IOException e) {
+            Logger.log(IStatus.ERROR, "Couldn't delete the log file!" + tempCompCmd.toString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Couldn't delete the log file!" appears 2 times in the file.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)